src/lib/utils/platform.js

```diff
@@ -43,6 +45,8 @@ export const isAndroid = isAndroidWeb || isAndroidNative
 // otherwise (e.g. running on native) will return false (because library wrongly returns true in that case)
 export const isBrowser = isWeb ? isBrowserWeb : false
 
+export const isEmulator = isMobileNative && DeviceInfo.isEmulator()
```

Comment:
```suggestion
export const isEmulator = isMobileNative ? DeviceInfo.isEmulator() : Promise.resolve(false)
```

Reply:
if isMobileNative is false, the result of expression will be `false` of the boolean type, not a promise